go.mod

```diff
@@ -28,6 +28,7 @@ require (
 	github.com/gophercloud/gophercloud v0.22.0
 	github.com/hooklift/gowsdl v0.5.0
 	github.com/infobloxopen/infoblox-go-client v1.1.1
+	github.com/infobloxopen/infoblox-go-client/v2 v2.1.1
```

Comment:
This has the old client still imported. Is it required?

Reply:
No, it is redundant. I will remove it in the next commit.

Reply:
Done.